R/po_create.R

```diff
@@ -0,0 +1,63 @@
+#' Create a new `.po` file
+#'
+#' @description
+#' `po_create()` creates a new `po/{languages}.po` containing the messages to be
+#' translated.
+#'
+#' Generally, we expect you to use `po_create()` to create new `.po` files
+#' but if you call it with an existing translation, it will update it with any
+#' changes from the `.pot`. See [po_update()] for details.
+#'
+#' @param languages Language identifiers. These are typically two letters (e.g.
+#'   "en" = English, "fr" = French, "es" = Spanish, "zh" = Chinese), but
+#'   can include an additional suffix for languages that have regional
+#'   variations (e.g. "fr_CN" = French Canadian, "zh_CN" = simplified
+#'   characters as used in mainland China, "zh_TW" = traditional characters
+#'   as used in Taiwan.)
+#' @inheritParams po_extract
+#' @export
+po_create <- function(languages, dir = ".", verbose = !is_testing()) {
+  package <- get_desc_data(dir, "Package")
+  po_files <- po_language_files(languages, dir)
```

Comment:
it might be nice to `warn()` here if e.g. R-pkg.pot is present but pkg.pot is not & the package has a `src/` directory.

OTOH, if the translation is handled entirely by `potools`, this should only happen if `src/` doesn't have any messages to translate, right? So maybe it will be a warning highly prone to false positives...

skipping for now, but file a follow-up issue if you think the warning is worthwhile.